bloxid/hashids.go

```diff
@@ -0,0 +1,102 @@
+package bloxid
+
+import (
+	"errors"
+
+	hashids "github.com/speps/go-hashids/v2"
+)
+
+const (
+	hashIDAllowedChar = "0123456789abcdef"
+	maxHashIDLen      = 36
```

Comment:
please add comment - this + prefix needs to be 40 chars (== DefaultUniqueIDDecodedCharSize)
https://github.com/infobloxopen/atlas-app-toolkit/pull/254/files

Reply:
perhaps define it as `DefaultUniqueIDDecodedCharSize - len(hashIDPrefix)`